src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,37 @@
-# TO-DO: Complete the selection_sort() function below 
-def selection_sort( arr ):
+# TO-DO: Complete the selection_sort() function below
+def selection_sort(arr):
     # loop through n-1 elements
-    for i in range(0, len(arr) - 1):
+    for i in range(0, len(arr)):
         cur_index = i
         smallest_index = cur_index
+        # print(arr)
         # TO-DO: find next smallest element
-        # (hint, can do in 3 loc) 
-             
-
-
-
-        # TO-DO: swap
-
-
-
+        # (hint, can do in 3 loc)
+        for j in range(cur_index+1, len(arr)):
+            if (arr[j] < arr[cur_index]):
+                smallest_index = j
+                # TO-DO: swap
+                temp = arr[smallest_index]
+                arr[smallest_index] = arr[cur_index]
+                arr[cur_index] = temp
```

Comment:
nice job on the selection, there is a one liner in python you can use, that i think looks cleaner which is like this:
a,b = b,a or in your case arr[small], arr[cur] = arr[cur], arr[small]